containments/mark2/contents/ui/main.qml

```diff
@@ -89,7 +89,7 @@ Item {
         anchors.fill: parent
         Kirigami.Theme.colorSet: Kirigami.Theme.Complementary
 
-        topPadding: panel.dragMargin
+  //      topPadding: panel.dragMargin
```

Comment:
ok for now, tough i stil lthink that's correct.. one of the reasons was the background with a top black bar, which i'll see if i can correct in abstractdelegate.cpp area